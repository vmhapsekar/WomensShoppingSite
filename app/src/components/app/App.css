.app-style {
    font-family: "myriad-pro", "Myriad Pro", Myriad, helvetica, arial, sans-serif;
    font-size: 16px;
    line-height: 1.42857;
    color: #8a909a;
    background-color: #fff;
    background-color: #EEEDED;
    height: auto;
    *:focus {
        outline: none !important;
    }
}