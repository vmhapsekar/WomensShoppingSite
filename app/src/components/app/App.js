import React, { Component } from 'react'
import {
  BrowserRouter as Router,
  Route
} from 'react-router-dom'
import {LANDING_URL, LAUNCH_URL} from '../../constants/appUrls'
import LandingContainer from '../home/landing/LandingContainer'
import './App.css'

export class App extends Component {
  render () {
    return (
      <Router>
        <div className='app-style'>
          <Route exact path={LAUNCH_URL} component={LandingContainer} />
          <Route exact path={LANDING_URL} component={LandingContainer} />
        </div>
      </Router>

    )
  }
}

export default App
