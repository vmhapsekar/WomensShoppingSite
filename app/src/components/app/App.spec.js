
import React from 'react'
import {App} from './App'
import { shallowToJson } from 'enzyme-to-json'
import Enzyme, { shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

// React 16 Enzyme adapter
Enzyme.configure({ adapter: new Adapter() })

global.requestAnimationFrme = function (callback) {
  setTimeout(callback, 0)
}

describe('<App> should load', () => {
  test('snapshot test', () => {
    const component = shallow(<App />)
    const tree = shallowToJson(component)
    expect(tree).toMatchSnapshot()
  })

  test('Should render Router, Routes and ModalRoot', function () {
    const component = shallow(<App />)
    const hashRoute = component.find('Router')
    expect(hashRoute).toBeDefined()
    const route = component.find('Route')
    expect(route).toBeDefined()
    expect(route).toHaveLength(2)
  })
})
