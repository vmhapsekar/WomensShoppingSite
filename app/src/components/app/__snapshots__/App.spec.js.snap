// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> should load snapshot test 1`] = `
<BrowserRouter>
  <div
    className="app-style"
  >
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
    <Route
      component={[Function]}
      exact={true}
      path="/womens-top"
    />
  </div>
</BrowserRouter>
`;
